ante/gov_vote_ante.go

```diff
@@ -82,22 +83,25 @@ func (g GovVoteDecorator) ValidateVoteMsgs(ctx sdk.Context, msgs []sdk.Msg) erro
 
 		enoughStake := false
 		delegationCount := 0
-		stakedTokens := sdk.NewDec(0)
+		stakedTokens := math.LegacyNewDec(0)
 		g.stakingKeeper.IterateDelegatorDelegations(ctx, accAddr, func(delegation stakingtypes.Delegation) bool {
 			validatorAddr, err := sdk.ValAddressFromBech32(delegation.ValidatorAddress)
 			if err != nil {
 				panic(err) // shouldn't happen
 			}
-			validator, found := g.stakingKeeper.GetValidator(ctx, validatorAddr)
-			if found {
-				shares := delegation.Shares
-				tokens := validator.TokensFromSharesTruncated(shares)
-				stakedTokens = stakedTokens.Add(tokens)
-				if stakedTokens.GTE(minStakedTokens) {
-					enoughStake = true
-					return true // break the iteration
-				}
+			validator, err := g.stakingKeeper.GetValidator(ctx, validatorAddr)
+			if err != nil {
```

Comment:
Now that I'm looking at this version of the diff, maybe we should handle the case when `err` is of type `stakingtypes.ErrNoValidatorFound` and not panic-ing in that case, just skip.